utils/constants.py

```diff
@@ -48,3 +48,4 @@ def _get_env():
 SKALE_VAL_ABI_FILE = os.path.join(SKALE_VAL_CONFIG_FOLDER, 'abi.json')
 
 WALLET_TYPES = ['software', 'ledger']
+DELEGATION_PERIOD_OPTIONS = ['3', '6', '9', '12']  # strings because of click.Choice design
```

Comment:
I think we should split actual constants and config parameters to different files